course_inventory/inventory.py

```diff
@@ -70,7 +70,8 @@ def make_request_using_api_utils(url: str, params: Union[Dict[str, Any], None] =
                 logger.info('Beginning next attempt')
 
     logger.error('The maximum number of request attempts was reached')
-    return response
+    logger.error('Data could not be gathered; the program will exit')
```

Comment:
will it make sense to have the URL output in the log message?

Reply:
The URL for the failed requests? Sure, yeah, I could add that.

Reply:
yes

Reply:
Okay, I think this was resolved. See this commit: https://github.com/tl-its-umich-edu/course-inventory/pull/177/commits/695d45f6e17b24988dfae84826f5d2fb4032473c